/patrol_node:
  ros__parameters:
    initial_point: [0.0, 0.0, 0.0]
    target_point: [
      0.0, 0.0, 0.0,
      1.0, 3.0, 3.14,
      -1.0, -1.0, 3.14,
      -5.0, -1.0, 3.14
    ]
    use_sim_time: True